docs/Using Fleet/MDM-migration-guide.md

```diff
@@ -132,7 +132,12 @@ spec:
 
 2. Fill in the above keys under the `mdm.macos_migration` key. 
 
-To learn about each option, in the Fleet UI, select the avatar on the right side of the top navigation, select **Settings > Integrations > Mobile device management (MDM)**, and scroll down to the **End user migration workflow** section.
+In configuring the end-user migration workflow, administrators have the option to set one of two migration modes:
+
+- `"voluntary"`: In this mode, end users will encounter a prompt within the Fleet Desktop menu where they can opt to initiate the migration by selecting "Migrate to Fleet." If a user is unenrolled from the previous MDM, they will receive this prompt at intervals of approximately 15-20 minutes.
+- `"forced"`: With this setting, end users will receive a prompt to begin the migration process at regular intervals of 15-20 minutes, ensuring persistent reminders to complete the migration.
+
+To find this option in the Fleet UI, select the avatar on the right side of the top navigation, select **Settings > Integrations > Mobile device management (MDM)**, and scroll down to the **End user migration workflow** section.
```

Comment:
@noahtalerman I didn't explain _why_ the interval because it was getting too wordy and I wasn't sure if it adds value. Happy to include it!

Reply:
@roperzh thanks for this. The changes look great. I agree the _why_ will add too many words.

That said, after looking at the doc changes, I'm now thinking updating the UI copy to say "...the window appears every 15-20 minutes" is better. It's short, accurate, and easy to understand.

![Screenshot 2023-09-26 at 3 56 33 PM](https://github.com/fleetdm/fleet/assets/47070608/b576c6c8-ea97-4258-8711-8d571306bf7a)

Also, this way, we can continue to point the users to the UI so the docs have less words and we don't have to maintain this information in the docs.

Sorry you sunk some time into this. If you agree, I'm happy to submit the PR to the copy in the UI.

Reply:
@noahtalerman that sounds good to me! thanks for the review, and please don't worry, happy to iterate.

I'll close this PR.